test/integration/tests/abrmd_policypassword.sh

```diff
@@ -32,6 +35,12 @@ cleanup "no-shutdown"
 
 echo "plaintext" > $plain_txt
 
+
+encryptdecrypt_supp=false
+if [ ! -z "$(tpm2_getcap commands | grep 'encryptdecrypt:')" ]; then
+  encryptdecrypt_supp=true
+fi
```

Comment:
Use a different command. Change the key type to rsa and do a sign and verify operation.

Reply:
Done.